web/README.md

```diff
@@ -10,6 +10,14 @@ yarn dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-## Related
+## Server API
 
 - [doc](https://book.prow.tidb.io/plugins/owners.html)
+
+The api design is just like `/repos/:org/:repo/pulls/:numbers/owners`.
+
+The api such as `https://prow.tidb.io/ti-community-owners/repos/pingcap/tidb-operator/pulls/3522/owners`
```

Comment:
Maybe you can combine two sentences together（just my suggestion）:

The API design is just like `/repos/:org/:repo/pulls/:numbers/owners`, such as `https://prow.tidb.io/ti-community-owners/repos/pingcap/tidb-operator/pulls/3522/owners`.

Reply:
ok~